packages/react-components/src/organisms/MainNavigation.tsx

```diff
@@ -6,28 +6,31 @@ import {
   vminLinearCalc,
   largeDesktopScreen,
   mobileScreen,
+  smallDesktopScreen,
 } from '../pixels';
 import { NavigationLink } from '../atoms';
 import { networkIcon, libraryIcon, newsIcon } from '../icons';
 
-const width = `max(258px, ${vminLinearCalc(
-  mobileScreen,
-  312,
-  largeDesktopScreen,
-  258,
-  'px',
-)})`;
-
 const containerStyles = css({
-  width,
+  minWidth: `max(${vminLinearCalc(
```

Comment:
James requested the change on small desktop, this is now essentially a V-shaped function, with the bottom point at small desktop

Reply:
I recently added a vminLinearCalcCapped function. Maybe it would make sense to extend this to support a minimum value in addition to a maximum value?

Reply:
TBH I'm not convinced that should exist. It's just a composition of `vminLinearCalc` and CSS `min`. Sometimes you want `min`, sometimes you want `max`, sometimes you want both, sometimes, like here, you want a `max` of *two* calcs. I would prefer removing that function as it's overly specific and its goal can be achieved by composition of functions

Reply:
@kmaid  further thoughts?

Reply:
Merging for now so that I submit a follow up for review. As I said I'd like to remove `vminLinearCalcPadded` entirely, maybe I'll open a PR+discussion for that at some point.